bench/micro/fiber_bench.ml

```diff
@@ -133,6 +133,29 @@ let%bench_fun "Fiber.parallel_map" =
       (Fiber.parallel_map l ~f:Fiber.return)
     |> ignore
 
+let pool_run tasks =
```

Comment:
Can you explain what this benchmarks exactly? I have a hard time following this code because of my poor familiarity with pools. Aren't you supposed to use `Pool.task` somewhere?

Reply:
I was staring at the results and I knew there was something missing. Good catch.

Reply:
Ah, right, this makes more sense now.